src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -104,3 +86,30 @@ export interface PlacementTarget {
 export const CLOUD_TIER = 'cloud-s3';
 
 export const DEFAULT_PLACEMENT = 'default-placement';
+
+export const ALLOW_READ_THROUGH_TEXT =
+  'Enables fetching objects from remote cloud S3 if not found locally (default: false).';
```

Comment:
Lets not say default value. It is pretty visible.
```suggestion
  'Enables fetching objects from remote cloud S3 if not found locally.';
```